c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -18,15 +18,15 @@ https://camptocamp.github.io/c2cgeoportal/2.0/integrator/legacy_cgxp.html
     cp CONST_create_template/{{package}}/templates/desktop.html {{package}}/templates/
     cp CONST_create_template/{{package}}/static-ngeo/less/desktop.less {{package}}/static-ngeo/less
     cp CONST_create_template/{{package}}/static-ngeo/js/desktop.js {{package}}/static-ngeo/js
-    cp CONST_create_template/{{package}}/static-ngeo/components {{package}}/static-ngeo/
+    cp -r CONST_create_template/{{package}}/static-ngeo/components {{package}}/static-ngeo/
 
 2. Remove the cgxp interfaces:
 
-    git rm {{package}}/templates/index.html
+    git rm -f {{package}}/templates/index.html
```

Comment:
Needed if you upgrade from 1.6 to 2.1 directly. Because changelog 2.0 ask to modify these files